frontend/catalyst/cuda/__init__.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains a CudaQDevice and the qjit
+entry point.
+"""
+
+from pathlib import Path
+
+import cudaq
+import pennylane as qml
+
+from catalyst.cuda.catalyst_to_cuda_interpreter import interpret
+
+
+def qjit(fn=None, **kwargs):
```

Comment:
@erick-xanadu pylint noticed that `**kwargs` is not used here 🤔 This means that user provided arguments will be 'eaten' without warning

Reply:
I'm raising an error. I left it unimplemented for now, we can go back. https://github.com/PennyLaneAI/catalyst/pull/477/commits/1645094c7bc47df4796146a55ef534ad275decb7